src/Transport.ts

```diff
@@ -148,9 +149,41 @@ export type PlainRtpParameters =
 	port: number;
 };
 
+export type TransportProduceParameters<AppData = unknown> =
```

Comment:
`AppData` default value should not be `unkown` but `Record<string, unknown>` (everywhere).